pkg/repositories/transformers/workflow_test.go

```diff
@@ -133,7 +156,20 @@ func TestFromWorkflowModels(t *testing.T) {
 		Name:         "name b",
 		Version:      "version b",
 	}, workflowList[1].Id))
+
+	// Expected to be nil
+	var workflowInterface1 core.TypedInterface
+	err = proto.Unmarshal(workflowModels[1].TypedInterface, &workflowInterface1)
+	assert.NoError(t, err)
+
 	assert.True(t, proto.Equal(&admin.WorkflowClosure{
 		CreatedAt: createdAtBProto,
+		CompiledWorkflow: &core.CompiledWorkflowClosure{
+			Primary: &core.CompiledWorkflow{
+				Template: &core.WorkflowTemplate{
+					Interface: &workflowInterface1,
```

Comment:
nit: I think you don't need to initialize this interface if it's just going to be nil anyways